src/search/search.component.html

```diff
@@ -34,12 +34,12 @@
 			class="bx--toolbar-search__btn"
 			(click)="openSearch()"
 			aria-label="Open search">
-			<svg ibmIcon="search" size="16" class="bx--search-magnifier"></svg>
+			<svg ibmIcon="search" size="16" class="bx--search-magnifier-icon"></svg>
 		</button>
 		<svg
 			ibmIcon="search"
 			*ngIf="tableSearch || !toolbar"
-			class="bx--search-magnifier"
+			class="bx--search-magnifier-icon"
```

Comment:
So for me I get this: 
![image](https://user-images.githubusercontent.com/1744185/118289228-1fbbc380-b4a3-11eb-8b65-0464b46d4a2c.png)

If I change it to this:
```suggestion
			class="bx--search-magnifier"
```
It fixes the position of the icon:
![image](https://user-images.githubusercontent.com/1744185/118289355-4417a000-b4a3-11eb-9302-8c0945de5723.png)

The toolbar search looks correct though :+1:

Reply:
hey @cal-smith - So i tested it with 
```suggestion
			class="bx--search-magnifier"
```
but it keeps the magnifier outside of search box, but with 
```suggestion
			class="bx--search-magnifier-icon"
``` 
it works for me. 

<img width="1046" alt="search-bar" src="https://user-images.githubusercontent.com/306189/123284193-10845880-d4da-11eb-8567-65f160c5f91f.png">

Reply:
Interesting ... I think the `carbon-components` version needs to be updated in this PR then (I believe @PPPiQ is working on that)